app/app.go

```diff
@@ -447,6 +461,17 @@ func New(
 		app.AccountKeeper, app.BankKeeper, scopedTransferKeeper,
 	)
 
+	icaMessageRouter := MessageRouterFunc(func(msg sdk.Msg) baseapp.MsgServiceHandler {
```

Comment:
feels like the exact same code is here too ```wasmMessageRouter := MessageRouterFunc(func(msg sdk.Msg) baseapp.MsgServiceHandler {
		return pioMsgFeesRouter.Handler(msg)
	})``` may just want one MsgRouter variable?

Reply:
Good point. I removed the duplicate router.